__tests__/integration.test.js

```diff
@@ -181,47 +181,107 @@ describe("POST: /api/articles/:article_id/comments", () => {
+      .send({
+        username: "Do_not_exist",
+        body: "hello",
+      })
+      .expect(400)
+      .then((response) => {
+        const { body } = response;
+        expect(body).toEqual({ message: "bad request" });
+      });
+  });
   test("POST: responds with a 404 status code no ID is provided", () => {
     return request(app)
-    .post("/api/articles//comments")
-    .send({
-      username: "Do_not_exist",
-      body: "hello"
-    })
-    .expect(404)
-    .then((response) => {
-      const {body} = response
-      expect(body).toEqual({message: "Path does not exist"})
-    })
-  })
+      .post("/api/articles//comments")
+      .send({
+        username: "Do_not_exist",
+        body: "hello",
+      })
+      .expect(404)
+      .then((response) => {
+        const { body } = response;
+        expect(body).toEqual({ message: "Path does not exist" });
+      });
+  });
+});
+describe("PATCH: /api/articles/:article_id", () => {
+  test("PATCH: responds with a 200 status code when updated votes by increment of 1", () => {
+    let newVote = 1;
+    const articleVotes = { inc_votes: newVote };
+    return request(app)
+      .patch("/api/articles/1")
+      .send(articleVotes)
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.articleVotes.votes).toBe(101);
+      });
+  });
+  test("PATCH: responds with a 200 status code when updated votes by decrement of 100", () => {
+    let newVote = -100;
+    const articleVotes = { inc_votes: newVote };
+    return request(app)
+      .patch("/api/articles/1")
+      .send(articleVotes)
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.articleVotes.votes).toBe(0);
+      });
+  });
+  test("PATCH: responds with a 404 when provided article_id which does not exist in the database", () => {
+    let newVote = -100;
+    const articleVotes = { inc_votes: newVote };
+    return request(app)
+      .patch("/api/articles/12345")
+      .send(articleVotes)
+      .expect(404)
+      .then((response) => {
+        const { body } = response;
+        expect(body).toEqual({ message: "article_id not found" });
+      });
+  });
+  test("PATCH: responds with a 400 when provided an invalid ID", () => {
+    let newVote = -100;
+    const articleVotes = { inc_votes: newVote };
+    return request(app)
+      .patch("/api/articles/hello-there")
+      .send(articleVotes)
+      .expect(400)
+      .then((response) => {
+        const { body } = response;
+        expect(body).toEqual({ message: "bad request" });
+      });
+  });
+    test("PATCH: responds with a 400 when provided an invalid vote value", () => {
+      let newVote = "this is great!";
+      const articleVotes = { inc_votes: newVote };
+      return request(app)
+        .patch("/api/articles/1")
+        .send(articleVotes)
+        .expect(400)
+        .then((response) => {
+          const { body } = response;
+          expect(body).toEqual({ message: "bad request" });
+        });
+    });
```

Comment:
Overall great happy path and error handling tests 🙌 , just need to check the whole article object in your happy path tests